src/features/AddPaginationBar/ui/PaginationBar.tsx

```diff
@@ -0,0 +1,81 @@
+import { FC, useEffect, useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { getPageCount, getSomePagination } from 'shared/lib/changeData';
+import { Paths } from 'shared/types';
+import './PaginationBar.css';
+
+interface PaginationProps {
+  totalResults: string | undefined;
+  activePage: number;
+}
```

Comment:
```suggestion
interface PaginationProps {
  totalResults?: string;
  activePage: number;
}
```
если сделать проп не обязательным, то можно убрать undefined